src/rgw/rgw_zone.h

```diff
@@ -568,7 +568,7 @@ struct RGWZone {
   bool sync_from_all;
   set<std::string> sync_from; /* list of zones to sync from */
 
-  RGWZone() : log_meta(false), log_data(false), read_only(false), bucket_index_max_shards(0),
+  RGWZone() : log_meta(false), log_data(false), read_only(false), bucket_index_max_shards(11),
```

Comment:
what about making 11 a constexpr?

Reply:
good point! turning this into a constexpr variable also allows us to document it